testing/apmsoak/README.md

```diff
@@ -0,0 +1,54 @@
+# Run soak test worker
+
+**NOTE: This is unlikely to be useful for non-Elastic employees**
+
+This folder provides tools to enable creation of a worker instance to send load to any APM Server. Worker run the `apmsoak` binary to generate a continuous load and since it is a long running process, it is run as systemd unit configured to restart on failure. Monitoring is also setup for the worker as part of the terraform module to monitor the node resources and the systemd unit status.
```

Comment:
```suggestion
This folder provides tools to enable creation of a worker instance to send load to any APM Server. The worker runs the `apmsoak` binary to generate a continuous load and since it is a long running process, it is run as systemd unit configured to restart on failure. Monitoring is also setup for the worker as part of the terraform module to monitor the node resources and the systemd unit status.
```